insights/parsers/pam.py

```diff
@@ -108,8 +108,8 @@ class PamSSHD(PamDConf):
 """
 
 from collections import namedtuple
-from .. import Parser, get_active_lines, parser
-from ..parsers import unsplit_lines
+from insights import Parser, get_active_lines, parser
+from insights.parsers import unsplit_lines, keyword_search
```

Comment:
You are correct that the tests are not consistent with use of relative paths (although they should be), but the majority if not all parsers and combiners do use relative paths.